service/raft/raft_group0.cc

```diff
@@ -427,13 +427,37 @@ future<> raft_group0::join_group0(std::vector<gms::inet_address> seeds, bool as_
     group0_log.info("server {} joined group 0 with group id {}", my_id, group0_id);
 }
 
+struct group0_members {
```

Comment:
Re commit message:

1. we just cannot proceed without ensuring that every node is on the same page, this is a fundamental assumption of the procedure. We must contact every node.
2. Every step already has a retry loop, so I'm not sure what you're complaining about in the commit message.

Reply:
Or we could put a retry loop inside `get_inet_addrs`, I guess.

Reply:
It is fixed, I did't collapse the commits - will do once we merge the first part.